src/test/regress/sql/pg13_with_ties.sql

```diff
@@ -0,0 +1,123 @@
+SHOW server_version \gset
+SELECT substring(:'server_version', '\d+')::int > 12 AS server_version_above_twelve
+\gset
+\if :server_version_above_twelve
+\else
+\q
+\endif
+
+
+CREATE TABLE with_ties_table (a INT, b INT, c INT);
+SELECT create_distributed_table('with_ties_table', 'a');
+INSERT INTO with_ties_table VALUES (10, 20, 1), (11, 21, 2), (12, 22, 3), (12, 22, 4), (12, 22, 5), (12, 23, 6), (14, 24, 7);
+
+-- test ordering by distribution column
+SELECT * FROM with_ties_table ORDER BY a OFFSET 1 FETCH FIRST 2 ROWS WITH TIES;
```

Comment:
can we add more ordering columns to SELECT's to prevent possible test flakiness in future

Reply:
or maybe reduce the output to just the ORDER BY column.